test/e2e/apimachinery/flowcontrol.go

```diff
@@ -334,15 +351,16 @@ func makeRequest(f *framework.Framework, username string) *http.Response {
 	return response
 }
 
-func testResponseHeaderMatches(f *framework.Framework, impersonatingUser, plUID, fsUID string) bool {
-	response := makeRequest(f, impersonatingUser)
-	if response.Header.Get(flowcontrol.ResponseHeaderMatchedFlowSchemaUID) != fsUID {
-		return false
+func verifyResponseHeaderMatch(f *framework.Framework, response *http.Response, plUIDWant, fsUIDWant string) {
```

Comment:
```suggestion
func verifyResponseHeaderMatch(response *http.Response, plUIDWant, fsUIDWant string) {
```

don't need the first argument anymore